draft-ietf-masque-h3-datagram.md

```diff
@@ -919,47 +627,33 @@ STREAM(44): HEADERS            -------->
   :scheme = https
   :path = /target.example.org/443/
   :authority = proxy.example.org:443
-  Sec-Use-Datagram-Contexts = ?1
+  Sec-UDP-Timestamps = ?1
 
            <--------  STREAM(44): HEADERS
                         :status = 200
-                        Sec-Use-Datagram-Contexts = ?1
+                        Sec-UDP-Timestamps = ?1
 
 STREAM(44): DATA               -------->
-  Capsule Type = REGISTER_DATAGRAM_CONTEXT
-  Context ID = 0
-  Datagram Format Type = UDP_PAYLOAD
-  Datagram Format Additional Data = ""
+  Capsule Type = REGISTER_UDP_WITH_TIMESTAMP_CONTEXT
```

Comment:
In this example, the client wants to send the datagram without the extension to make sure that its first packet gets through to the target. Same for the next example below

Reply:
@DavidSchinazi I've updated the examples—now it just has a "supported extension" and "unsupported extension" case.

Reply:
but if this is the support casem then wouldn't the server reply with `Encoded Stream ID = 23` and `Payload = Encapsulated UDP Payload With Timestamp`?

Reply:
My comment still stands: the client is trying to send without the extension in its first flight

Reply:
It can! It was just getting long. I'll fix.

Reply:
Fixed